backend/app/tests/routers/test_routers.py

```diff
@@ -1,18 +1,19 @@
 from fastapi.testclient import TestClient
 
 from app.main import app
+from app.models.types.record_type import RecordType
 
 OK_STATUS_CODE = 200
 
 
-def test_next_records(record_key: str, record_type: str) -> None:
+def test_next_records(record_key: str, record_type: RecordType) -> None:
     """
     Test the /next_records endpoint.
     """
     with TestClient(app) as client:
         response = client.get(
-            url="/ap1/v1/nerdswipe/next_records",
-            params={"record_key": record_key, "record_type": record_type},
+            url="/api/v1/nerdswipe/next_records",
```

Comment:
Should probably be `/api/v1/records/next` ?

Reply:
GET is supposed to be idempotent, so this should probably be POST.

Reply:
I stuck with your initial suggestion (i.e. `/api/v1/next_records`).